modules/raycast/config.py

```diff
@@ -1,9 +1,13 @@
 def can_build(env, platform):
-    # Depends on Embree library, which only supports x86_64 and arm64.
-    if platform == "windows":
-        return env["arch"] == "x86_64"  # TODO build for Windows on ARM
-
-    return env["arch"] in ["x86_64", "arm64"]
+    # Supported architectures depend on the Embree library.
+    # No ARM32 support planned.
+    if env["arch"] == "arm32":
+        return False
+    # x86_32 only seems supported on Windows for now.
+    if env["arch"] == "x86_32" and platform != "windows":
+        return False
+    # The rest works, even wasm32!
+    return True
```

Comment:
I had forgotten about `rv64` and co. when writing this, and `ppc`.

Should likely readd something like in `3.x`:
```
    if env["arch"].startswith("rv") or env["arch"].startswith("ppc"):
        return False
```

Reply:
It would probably be better to invert the logic here, and list the architectures we *do* support. So:

```py
if env["arch"] in ["x86_64", "arm64", "wasm32"]:
    return True
if env["arch"] == "x86_32" and platform != "windows":
    return True
return False
```